### PR TITLE
MSDK-6656 - Update MSDK to v9.0.1

### PR DESCRIPTION
**Pull request by SalarZ**
- Update MSDK to v9.0.1
- Add localizations for Polish, Dutch, and English (United Kingdom)

### CODE REVIEW

File: IntegrationExample/Resources/Localizations/en-GB.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* 
+  Localizable.strings
+  IntegrationExample
+
+  Created by Hozefa Indorewala on 17.08.23.
```
Comment (hozefa26) [inline IntegrationExample/Resources/Localizations/en-GB.lproj/Localizable.strings:5]: how are they created by me and in August? 😅

**Review by hozefa26 — APPROVED** [review]
🚀 
please create a 3.0.0 release after merging this ticket.

File: IntegrationExample/Resources/Localizations/en-GB.lproj/Localizable.strings (hunk 1 above)
Comment (SalarZ) [reply IntegrationExample/Resources/Localizations/en-GB.lproj/Localizable.strings:5]: I think it uses the Base Localizable.strings file. Do I need to change them?

Comment (hozefa26) [reply IntegrationExample/Resources/Localizations/en-GB.lproj/Localizable.strings:5]: no i was just wondering 😄

**Pull request merged**